cacerts/BUILD

```diff
@@ -1,6 +1,15 @@
-package(default_visibility = ["//visibility:public"])
+package(default_visibility = ["//:__subpackages__"])
 
-sh_binary(
-    name = "extract_certs",
-    srcs = ["extract.sh"],
-)
+load("//:distro.bzl", "DISTRO_PACKAGES", "DISTRO_SUFFIXES")
```

Comment:
@chanseokoh not so sure about this approach. I've tried adding the distro.bzl at the root to avoid writing the same basic loop setup for deb9 and deb10 all over the place.

There are other places where the same loop is still present; but I thought it better to validate the idea first before going all.

Reply:
I personally think this seems smart! I hacked the Debian10 support into distroless in a fairly ... unplanned way. I think this seems a bit more structured.